lib/parse.ts

```diff
@@ -52,7 +66,7 @@ function toRequestOptions (options) {
     url = addQueryParameters(url, remainingParameters)
   } else {
     if ('data' in remainingParameters) {
-      body = remainingParameters.data
+      body = (remainingParameters as any).data
```

Comment:
Is there a better type than `any` here? `remainingParameters` Is always an object with `string` keys, the values can be anything though

Reply:
I have pushed a commit to fix this